test/lib/tasks/local_service_test.rb

```diff
@@ -0,0 +1,10 @@
+require "test_helper"
+
+class LocalServiceTaskTest < ActiveSupport::TestCase
+  context ".call" do
+    should "send the from and to LGSL codes to the change LGSL code service" do
+      ChangeLgslCodeService.expects(:call).with(1, 2)
+      ChangeLgslCodeService.call(1, 2)
```

Comment:
Oh by the way this test seems a bit bizarre I'm wondering if you didn't finish what you were doing? It doesn't seem to test a task at all and just expects a function call that you then make on the subsequent line.